R/new_matrixfit.R

```diff
@@ -489,8 +489,13 @@ make_parind <- function (parlist, corr_matrix_size, length_t, summands = 1) {
   n_corr <- corr_matrix_size
   ## parind is the index vector for the matrix elements
   parind <- array(1, dim = c(n_corr*length_t, 2))
+  parind_aux <- array(1, dim = c(n_corr*length_t, 2))
```

Comment:
This fixes the cases for a signal plus *one* pollution. What about having multiple pollutions? That is what the `summands` parameter was thought for.